source/clear-linux/guides/network/dpdk.rst

```diff
@@ -1,337 +1,347 @@
-   OS for Intel Architecture currently supports this action. You can use the
-   following commands::
+#. Unbind the NICs from the host to do pass-through with virtual machines. |CLOSIA|
+   supports this action. The commands take the format:
+
+   .. code-block:: bash
 
       echo "vendor device_ID" > /sys/bus/pci/drivers/pci-stub/new_id
       echo "entry for device" > /sys/bus/pci/drivers/igb/unbind
       echo "entry for device" > /sys/bus/pci/drivers/pci-stub/bind
       echo "vendor device_ID" > /sys/bus/pci/drivers/pci-stub/remove_id
 
+   Here is a working example:
+
    .. code-block:: bash
 
-      $ echo "8086 1521" > /sys/bus/pci/drivers/pci-stub/new_id
-      $ echo "0000:03:00.0" > /sys/bus/pci/drivers/igb/unbind
-      $ echo "0000:03:00.0" > /sys/bus/pci/drivers/pci-stub/bind
-      $ echo "8086 1521" > /sys/bus/pci/drivers/pci-stub/remove_id
+      sudo echo "8086 1521" > /sys/bus/pci/drivers/pci-stub/new_id
+      sudo echo "0000:03:00.0" > /sys/bus/pci/drivers/igb/unbind
+      sudo echo "0000:03:00.0" > /sys/bus/pci/drivers/pci-stub/bind
+      sudo echo "8086 1521" > /sys/bus/pci/drivers/pci-stub/remove_id
+
+#. Assign the unbound NICs to the KVM virtual machine (guest).
+   Modify the :file:`start_qemu.sh` script in `qemu-system-x86_64` arguments, and
+   add the lines with the host's NICs information in the format:
 
-#. Assign to the KVM virtual machine (guest) the unbound NICs previously noted.
-   Modify the ``start_qemu.sh`` script in ``qemu-system-x86_64`` arguments, and
-   add the lines with the host's NICs information::
+   .. code-block:: bash
 
-   -device pci-assign,host="<entry for device>",id=passnic0,addr=03.0
-   -device pci-assign,host="<entry for device>",id=passnic1,addr=04.0
+      -device pci-assign,host="<entry for device>",id=passnic0,addr=03.0
+      -device pci-assign,host="<entry for device>",id=passnic1,addr=04.0
 
-   A working example:
+   Here is a working example:
 
    .. code-block:: bash
 
       -device pci-assign,host=03:00.0,id=passnic0,addr=03.0 \
       -device pci-assign,host=03:00.3,id=passnic1,addr=04.0 \
 
-#. If you would like to add more NUMA machines to the virtual machine, you can
-   add the next line in the Makefile boot target::
+#. Add more NUMA machines to the virtual machine by adding lines to the
+   Makefile boot target in the format:
+
+   .. code-block:: bash
 
       -numa node,mem=<memory>,cpus=<number of cpus>
 
-   As a working example for a virtual machine with 4096 of memory and four CPUs, the configuration
-   would look like this::
+   Here is a working example for a virtual machine with 4096 memory and four
+   CPUs:
+
+   .. code-block:: bash
 
     -numa node,mem=2048,cpus=0-1 \
     -numa node,mem=2048,cpus=2-3 \
 
-   This means that each NUMA machine has to use the same quantity of memory.
+   .. note:: Each NUMA machine must use the same quantity of memory.
 
-#. Finally, run the ``start_qemu.sh`` script.
+#. Run the :file:`start_qemu.sh` script.
```

Comment:
Can someone provide an output example here? Unclear if script output is relevant for users, please reject if it's not.